paper-server/src/main/java/io/papermc/paper/PaperServerInternalAPIBridge.java

```diff
@@ -12,4 +21,17 @@ public class PaperServerInternalAPIBridge implements InternalAPIBridge {
     public DamageEffect getDamageEffect(final String key) {
         return CraftDamageEffect.getById(key);
     }
+
+    @Override
+    public CombatEntry createCombatEntry(LivingEntity entity, DamageSource damageSource, float damage) {
+        net.minecraft.world.entity.LivingEntity mob = ((CraftLivingEntity) entity).getHandle();
+        net.minecraft.world.damagesource.FallLocation fallLocation = net.minecraft.world.damagesource.FallLocation.getCurrentFallLocation(mob);
+        float fallDistance = mob.fallDistance;
+        return new PaperCombatEntryWrapper(new net.minecraft.world.damagesource.CombatEntry(((CraftDamageSource) damageSource).getHandle(), damage, fallLocation, fallDistance));
```

Comment:
can you just call the `createCombatEntry` overload here.
Makes life easier if that ctor ever changes.